src/app/debug/table/table.component.ts

```diff
@@ -205,9 +205,7 @@ export class TableComponent implements OnInit, OnDestroy {
         value = '';
       }
     }
-    this.tableSettings.currentFilters = new Map<string, string>();
-    this.tableSettings.currentFilters = filters;
-
+    this.tableSettings.currentFilters = new Map(filters);
```

Comment:
Why create a new `Map` object? 
`filters` already is a `Map`?
```suggestion
    this.tableSettings.currentFilters = filters;
```

Reply:
This was for debugging and forgot to get rid of it. The value in activeFilters wasn't being updated and this was one of the debugging lines for hard updating it. I'll change it back.